src/main/webapp/constants/constant.ts

```diff
@@ -1,3 +1,3 @@
 export default {
     FILE_MAX_SIZE: '500',
-};
+} as const;
```

Comment:
why do we need those assertions?

Reply:
Regular:
<img width="460" alt="image" src="https://github.com/splunk/addonfactory-ucc-base-ui/assets/142901247/33f3ef96-223d-4d27-a792-c949e8d62f28">

with `as const`
<img width="455" alt="image" src="https://github.com/splunk/addonfactory-ucc-base-ui/assets/142901247/8436a1a0-9bb4-4dfb-8a3f-723c1ee955ca">

Maybe it does not bring any value exactly here, but it will make sense if we will as some other values.